src/css/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$primary-green: #C4EB67;
+$secondary-green: #359567;
+$main-and-button-font: "Quicksand", sans-serif;
+$h1-font: "Sulphur Point", sans-serif;
```

Comment:
This has a thing that says "no newline at end of file" next to it, any idea what that means?

Reply:
I read some stuff online about it, but I'm still unclear what purpose it serves. The file on my end doesn't have any weirdness going on. From what I've read I'm assuming some languages require a new line after the last entry, but I'm not exactly sure.

Reply:
Ok, good to know!